test/unit/write-transforms.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash';
+
+/**
+ * Definitions to swap on the native result. Key is the new string and Value is the pattern to replace
+ *
+ * @type {Object.<string, RegExp>}
+ */
+export const nativeToPostscribe = [_.trim];
```

Comment:
I don't understand what's going on here.  You've declared the type as Object, but it is setup as an array with `_.trim`.